app/components/moderation-list/component.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+/**
+ * Provides a list of pending, accepted, and rejected preprints. Provides filtering by preprint state and sorting based
+ * on the preprint creation date. If number of records in each type exceeds 10, pagination is enabled. Current page for
+ * each state is tracked and used by the paginator using the added pager service . Pagination is using the
+ * pagination-pager addon (source: https://github.com/knownasilya/pagination-pager).
```

Comment:
Currently, pagination-pager is being used by registries and preprints. [osf-paginator](https://github.com/CenterForOpenScience/ember-osf/tree/develop/addon/components/osf-paginator ) is not used.